tensorboard/components/tensor_widget/README.md

```diff
@@ -0,0 +1,21 @@
+# Tensor Widget
+
+Tensor Widget is a DOM library for flexible, interactive, and efficient
+display of values of tensors, i.e., typed n-dimensional arrays.
+
+Features
+
+- The header region displays the dtype, shape and optionally the name of
+  the tensor.
+- The **health pill** section of the header displays succinct summary of the
```

Comment:
Health pill has been in use at least in the context of TF Debugger since early 2017. It can be argued that the medical connotation is meaningful, because the UI can be used to diagnose the presence of "pathological values" in the tensors such as Infinity and NaN. But the shape of the indicator is also relevant, as you pointed out.

Reply:
Yes, I realize it's been in use before now, I just figured that this is broadening the reach of the term substantially and perhaps it was worth mentioning.  A "pill" in a medical context is a treatment tool more typically than a diagnosis tool which is why it didn't really communicate to me that it was an indicator, as opposed to e.g. something like "health indicator" or "health check".  But if you feel this is the best wording it's your call.

Reply:
IMO, the risk of confusing this with a fix (i.e., analogous to a medical treatment) is low, because whether it's TensorBoard or a debugger, most users should understand it's a tool primarily for observation, not intervention. To fix something, one would have to  go to the code editor and change the code there. So I'd keep this terminology. It's easy to remember and registers well with the shape of the UI.